src/pages/ProductDetail/FavoriteModal.js

```diff
@@ -0,0 +1,168 @@
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+import FavoriteBtn from './FavoriteBtn';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBookmark as farBookmark } from '@fortawesome/free-regular-svg-icons';
+
+const FavoriteModal = ({
+  title,
+  productName,
+  productImg,
+  productSizes,
+  confirmText,
+  onConfirm,
+  visible,
+}) => {
+  useEffect(() => {
+    document.body.style.cssText = `
+      position: fixed; 
+      top: -${window.scrollY}px;
+      overflow-y: scroll;
+      width: 100%;`;
+    return () => {
+      const scrollY = document.body.style.top;
+      document.body.style.cssText = '';
+      window.scrollTo(0, parseInt(scrollY || '0', 10) * -1);
+    };
+  }, []);
+
+  if (!visible) return null;
+  return (
```

Comment:
```suggestion
  if (!visible) return null;

  return (
```

다른 문맥들끼리는 공백을 통해서 가독성에 신경써주세요

Reply:
수정완료